components/waitwhile/actions/import-customers/import-customers.ts

```diff
@@ -0,0 +1,13 @@
+export default {
+  name: "Import Customers",
+  version: "0.0.1",
+  key: "import-customers",
```

Comment:
This component looks unfinished?
```suggestion
  key: "waitwhile-import-customers",
```